index.html

```diff
@@ -135,6 +138,9 @@
           -o-transition: background-color 500ms ease-out;
           transition: background-color 500ms ease-out;
       }
+      .select2-container--default .select2-selection--multiple .select2-selection__choice {
```

Comment:
Already present 
https://github.com/org-roam/org-roam-server/blob/master/index.html#L143-L145

Reply:
The reason why I did this was because a white text looked better and more consistent with the other text on the page because they are all white. We can merge it to https://github.com/org-roam/org-roam-server/blob/a42a3559cf51ba6b7032c02ae07bf361a462f599/index.html#L143-L145

like so. 
``` js
      .darkmode .select2-container--default .select2-selection--single,
      .darkmode .select2-container--default .select2-selection--multiple,
      .darkmode .select2-search--dropdown,
      .darkmode .select2-results,
      .darkmode .select2-container--default .select2-selection--multiple .select2-selection__choice,
      .select2-container--default .select2-selection--multiple .select2-selection__choice {
          background-color: #1d1e20;
          color: #FFFFFF;
      }
```
I think it will be better to leave them separate for manageability sake in the future. It will add confusion later on if merged into one css block. `background-color:: #1d1e20;` won't apply to `selection__choice` because it is being set to green or red but this may change in the future.